### PR TITLE
Add comment ENV NEXT_SHARP_PATH  in Dockerfile

### PR DESCRIPTION
**Pull request by Insik-Han**
## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [ ] Make sure the linting passes by running `pnpm lint`
- [ ] The examples guidelines are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples)

### CODE REVIEW

File: examples/with-docker/Dockerfile [hunk 1]
```diff
@@ -37,6 +37,8 @@ WORKDIR /app
 ENV NODE_ENV production
 # Uncomment the following line in case you want to disable telemetry during runtime.
 # ENV NEXT_TELEMETRY_DISABLED 1
+# Uncomment the following line in case you use sharp library
+# ENV NEXT_SHARP_PATH /app/node_modules/sharp
```
Comment (ijjk) [inline examples/with-docker/Dockerfile:41]: Hi, this shouldn't be needed unless the sharp path is being installed in a separate docker layer, if it's installed as a dependency on the project it should work fine without the env variable.

**Comment by ijjk** [comment]
Gonna close this per above, potentially we could document this env variable in our image optimizer docs so feel free to add that if you'd like!

File: examples/with-docker/Dockerfile (hunk 1 above)
Comment (insik-han) [reply examples/with-docker/Dockerfile:41]: Thank you for your comment!
When I deploy to AWS Copilot, It doesn't work sharp. I agree to not need env variable comment but We need to check It dose work sharp when Deploy to docker with standalone.

Sorry, My english is not well

Comment (ijjk) [reply examples/with-docker/Dockerfile:41]: If `sharp` is installed in the project it should pick it up correctly, are you customizing how the dependencies are installed?

Comment (insik-han) [reply examples/with-docker/Dockerfile:41]: I try to docker locally. It did work.
I think It's problem of AWS Copilot

Here's my setting

https://github.com/insik-han/next-ts/blob/main/package.json

**Pull request closed without merging**